config/default_records/submission_templates/007_novaseqx_sequencing.yml

```diff
@@ -118,6 +118,13 @@ Limber-Bespoke - PCR - NovaSeqX paired end sequencing:
     product_line_name: Bespoke
     product_catalogue_name: GenericPCR
     request_type_keys: ["limber_pcr_bespoke", "limber_multiplexing", "illumina_htp_novaseqx_paired_end_sequencing"]
+Limber-Bespoke - PCR HiC - NovaSeqX paired end sequencing:
+  name: "Limber-Bespoke - PCR HiC - NovaSeqX paired end sequencing"
+  submission_class_name: "LinearSubmission"
+  related_records:
+    product_line_name: Bespoke
+    product_catalogue_name: GenericPCR
+    request_type_keys: ["limber_pcr_bespoke", "limber_multiplexing_hic", "illumina_htp_novaseqx_paired_end_sequencing"]
```

Comment:
Did you get a response from Danni as to whether this 'PCR HiC' naming made sense to her? I'm wondering whether just 'HiC' would be better.

Reply:
Also could consider not propagating the 'Bespoke' terminology, since the Bespoke team doesn't exist any more.

Reply:
And finally...! Do they want to do any other types of sequencing or just NovaseqX?

Normally we'd create a set of submission templates, including:
- One without any sequencing
- One for each sequencing instrument (e.g. NovaSeq 6000 and MiSeq)

Reply:
PCR came from the variant of the pipeline it's copied from, there were PCR and no PCR variants. I think it's important info, but Danni would know what's acceptable.
Same for Bespoke in that it came from the original pipeline name. This is the HiC variant so same name with HiC in it.

Reply:
Yeah I have requested some info on the submission name [here](https://github.com/sanger/limber/issues/1958#issuecomment-2435467328) but I still haven't got any reply for it.